### ISSUE TITLE
Article gets "lost" when refreshing while viewing article

### ISSUE THREAD

**Issue by The-Compiler** [akrennmair/newsbeuter#265]
This is on 03d3b2d as well, with ttrss.

When viewing an article while newsbeuter is reloading, sometimes I see this:

```
newsbeuter 2.10 - Article '' (4294967295 unread, 0 total)
Date: Thu, 01 Jan 1970 01:00:00 +0100

~
[...]
```

Ctrl-L doesn't change anything, but going back (to the article list) and forward (to the article) solves the issue.

I thought I've seen another bug report about this in the past, but I can't find it.

**Comment by The-Compiler** [comment]
Sometimes I also get a completely corrupted screen:


![newsbeuter](https://cloud.githubusercontent.com/assets/625793/12909890/b4e4b1f8-cf05-11e5-8292-c33ddf89ea80.png)

In this case, I think the feed I was viewing an article from was refreshed, and a new article was inserted into the same feed.

**Comment by Minoru** [comment]
> I thought I've seen another bug report about this in the past, but I can't find it.

Yeah, here it is: #223 Post further details—if any—there; I'll close this issue.

> Sometimes I also get a completely corrupted screen:

That's new! Let's see if it gets fixed with fix to #223, and if not, we'll split that out into separate bug.
